app/components/windows/EditStreamInfo.vue

```diff
@@ -30,11 +30,20 @@
           <div class="input-label"/>
           <div class="input-wrapper">
             <div class="checkbox">
-              <input
-               type="checkbox"
-               v-model="useOptimizedProfile"
-              />
-              <label><span>Use optimized encoder settings for this game</span></label>
+              <div v-if="!genericProfiles">
+                <input
+                type="checkbox"
+                v-model="useOptimizedProfile"
```

Comment:
You're going to want to indent `type` and `v-model` an additional level.  Apparently prettier auto formatting for vue templates is coming soon.